Makefile

```diff
@@ -66,11 +66,15 @@ lint:
 vet:
 	go vet $(PKGS)
 
+simple:
+	go get -u honnef.co/go/tools/cmd/gosimple
+	gosimple $(PKGS)
```

Comment:
You might want to split out the `go get -u honnef.co/go/tools/cmd/gosimple` into a separate make-rule, e.g.:

```
$(GOPATH)/bin/gosimple:
        go get -u honnef.co/go/tools/cmd/gosimple

simple: $(GOPATH)/bin/gosimple
	$(GOPATH)/bin/gosimple $(PKGS)
```

.. that way you won't keep recompiling/updating the "gosimple" binary (e.g. the "gosimple" utility will get built only when missing).